src/codegen/compile.ml

```diff
@@ -3804,7 +3813,7 @@ module BigNumLibtommath : BigNumType = struct
   let truncate_to_word32 env = E.call_import env "rts" "bigint_to_word32_wrap"
   let truncate_to_word64 env = E.call_import env "rts" "bigint_to_word64_wrap"
 
-  let from_word30 env = E.call_import env "rts" "bigint_of_word32"
```

Comment:
Interesting that this ever worked. Probably it is/was never used for a signed number?

Reply:
Would this not have worked? I think it's ok, but probably unreachable anyway (since a word30 would always be compact).